lte/gateway/python/scripts/subscriber_cli.py

```diff
@@ -123,6 +137,17 @@ def create_parser():
         cmd.add_argument('--lte-auth-opc', help='LTE authentication opc')
         cmd.add_argument('--lte-auth-next-seq', type=int,
                          help='LTE authentication seq number (hex digits)')
+        cmd.add_argument('--num-apn', type=int,
+                         help='Number of APNs to be configured')
+        cmd.add_argument('--apn-name1',
+                         help='Name of the 1st APN (ims/internet)')
+        cmd.add_argument('--qci1', type=int,
+                         help='QCI for the 1st APN')
+        cmd.add_argument('--apn-name2',
+                         help='Name of the 2nd APN (ims/internet)')
+        cmd.add_argument('--qci2', type=int,
+                         help='QCI for the 2nd APN')
+
```

Comment:
Hardwiring two APNs and then including num-apn as a paramter do not make much sense. Please see my previous comment on instead making APN configuration as an additional subcommand to add/delete/modify APNs.